src/Core/Shop/LogoUploader.php

```diff
@@ -143,6 +172,11 @@ public function update($fieldName, $logoPrefix, array $files = [])
         return false;
     }
 
+    public function isSvgMimeType(string $mimeType): bool
```

Comment:
This method should be in `ImageManager` or another class, it's not related to the Upload, but to the image itself

Reply:
Why not in the Validator Adapter ?

Reply:
We should avoid using one class to validate the whole world.
This one is related to an `Image` object

Reply:
@PierreRambaud I've moved isSvgMimeType in ImageManager and rebased, please review this PR is taking too much time, I'm tired.